### PR TITLE
Remove bdn-nogateway

### PR DESCRIPTION
**Pull request by brtkwr**

### CODE REVIEW

File: etc/p3-config/p3-config.yml [hunk 1]
```diff
@@ -586,7 +569,7 @@ p3_image_fedora:
 
 # Build of OpenHPC on CentOS
 p3_image_openhpc:
-  name: "CentOS7.4-OpenHPC"
+  name: "CentOS7.5-OpenHPC"
```
Comment (markgoddard) [inline etc/p3-config/p3-config.yml]: What's this? I think we just merged that change

Comment (brtkwr) [reply etc/p3-config/p3-config.yml]: Oh yeah!

Comment (brtkwr) [reply etc/p3-config/p3-config.yml]: I've removed them.

**Review by markgoddard — APPROVED** [review]
nyce1

**Pull request merged**